src/main.rs

```diff
@@ -194,9 +195,27 @@ pub struct NewOptions {
     pub template: new::Template,
 }
 
+use serde::{Deserialize, Serialize};
+use strum_macros::{Display, EnumString, EnumVariantNames};
+
+#[derive(Debug, Serialize, Deserialize, Display, EnumString, EnumVariantNames, Clone, Copy)]
+#[strum(serialize_all = "lowercase")]
+pub enum Target {
+    Erlang,
+    JavaScript
+}
+
 #[derive(StructOpt, Debug)]
 #[structopt(flatten)]
 pub struct CompilePackage {
+    #[structopt(
+        help = "The compilation target for the generated project", 
+        long = "target", 
+        possible_values = &Target::VARIANTS,
```

Comment:
`possible_values` is a clap method, StructOpt should deal with it automatically, so this line is not needed